src/utilities/page.ts

```diff
@@ -1,7 +1,10 @@
+import { pushEvent } from "./googleAnalytics";
+
 /**
  * Allows you to set the suffix of the page title
  * @param text - the suffix to set on the page title
  */
 export const setPageTitle = (text: string) => {
-  document.title = `Voltz Protocol ⚡ The Interest Rate Swap AMM${text ? ` - ${text}` : ''}`;
+  document.title = `Voltz ⚡ ${text ? `${text}` : ''}`;
+  pushEvent("path_change", text);
```

Comment:
proposal: lets change this event name to be title_change.

Let's add a track page change event (path_change) inside the App.tsx. We can use the API offered to us by react-router. Ping me for more details :D internet is slow...

Reply:
changed event name at [commit](https://github.com/Voltz-Protocol/voltz-ui/pull/268/commits/9bb0958572bdde864d23200e7f12fd6c19910057). Let's talk about actual path change, this would be very useful